cmd/hamctl/command/root.go

```diff
@@ -48,7 +48,6 @@ func NewCommand(version *string) (*cobra.Command, error) {
 			c.HelpFunc()(c, args)
 		},
 	}
-	command.AddCommand(NewPromote(&client, &service))
```

Comment:
The changes to `cmd/hamctl/command/promote.go` doesn't really make sense when we don't have the command anymore. Maybe we should just remove the file completely for now like on the server?

Reply:
This was actually a mistake. Had readded this 🤔